addons/website_crm_partner_assign/views/crm_lead_views.xml

```diff
@@ -184,15 +184,11 @@
                 <field name="user_id" position="after">
                     <field name="partner_assigned_id"/>
                 </field>
-                <xpath expr="//span[@t-esc='record.partner_id.value']" position="replace">
-                    <span class="o_text_overflow"
-                        t-esc="record.partner_id.value"
-                        t-att-title="record.partner_id.value"
-                        t-if="record.partner_id.value"/>
+                <xpath expr="//field[@name='partner_id']" position="replace">
+                    <field class="o_text_overflow" name="partner_id" t-att-title="record.partner_id.value" t-if="record.partner_id.value"/>
```

Comment:
Does t-att-title work on a field?
t-if="..." could be simplified in t-if="record.partner_id" I think

Reply:
No t-att-title not work on the field,I missed it here. So, I wrapped it in the span. i think we don't need that if condition now so i have removed it. 
Thank you for the suggestions

Reply:
In fact, I have observed yesterday that t-att-title can be used on a field (at least in a kanban-card).

Reply:
Why can you drop the t-if?